packages/SwingSet/src/vats/vat-admin/vat-vat-admin.js

```diff
@@ -121,6 +121,25 @@ export function buildRootObject(vatPowers) {
         pendingUpgrades.set(upgradeID, upgradeRR);
         return upgradeCompleteP;
       },
+      changeOptions(options) {
+        for (const option of Object.getOwnPropertyNames(options)) {
+          const value = options[option];
+          switch (option) {
+            case 'reapInterval':
+              assert(
+                value === 'never' || isNat(value),
+                'invalid reapInterval value',
```

Comment:
let's include the invalid value in the `assert` message, that will help with debugging